test/e2e/common/empty_dir.go

```diff
@@ -301,14 +301,15 @@ const (
 	volumeName    = "test-volume"
 )
 
-func doTestSetgidFSGroup(f *framework.Framework, image string, medium v1.StorageMedium) {
+func doTestSetgidFSGroup(f *framework.Framework, uid *int64, medium v1.StorageMedium) {
 	var (
 		filePath = path.Join(volumePath, "test-file")
 		source   = &v1.EmptyDirVolumeSource{Medium: medium}
-		pod      = testPodWithVolume(testImageRootUid, volumePath, source)
+		pod      = testPodWithVolume(uid, volumePath, source)
```

Comment:
strange thing is in the previous code, the passed image is not used. it always use testImageRootUid. That means testImageNonRootUid image is not tested at all?

Reply:
Indeed, this function ignores the given ``image``, which is a bug. But in other functions, like ``doTestSubPathFSGroup``, it uses the given ``image``, ``testImageNonRootUid`` was used by other tests.